pytorch_lightning/trainer/trainer.py

```diff
@@ -562,6 +567,10 @@ def from_argparse_args(cls, args):
         params = vars(args)
         return cls(**params)
 
+    @property
+    def show_progress_bar(self) -> bool:
+        return self.progress_bar_refresh_rate >= 1
```

Comment:
```suggestion
        warnings.warn("Attribute `show_progress_bar` is now set by `progress_bar_refresh_rate` since v0.7.1"
                          " and this attribute will be removed in v0.9.0", DeprecationWarning)
        return self.progress_bar_refresh_rate >= 1
```